extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/CommandListenerBuildItem.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.mongodb.deployment;
+
+import java.util.List;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+public final class CommandListenerBuildItem extends SimpleBuildItem {
```

Comment:
Can you please add some Javadoc to this class? This is necessary for the All Build Items page